stylelint-config/rules/use-defined-css-var-when-available/props-to-allowed-vars.js

```diff
@@ -0,0 +1,129 @@
+const SPACINGS = [
+  "--spacing-xs",
+  "--spacing-small",
+  "--spacing-medium",
+  "--spacing-large",
+  "--spacing-xl",
+  "--spacing-xxl",
+  "--spacing-xxxl",
+];
+
+const BOX_SHADOWS = ["--box-shadow-small", "--box-shadow-medium", "--box-shadow-large"];
+
+const BORDER_RADIUSES = ["--border-radius-small", "--border-radius-medium", "--border-radius-big"];
+
+const FONT_SIZES = [
+  // since "--font-size-10" and "--font-size-20" have the same value, we need to pick only one of them as "valid value", to allow autofix
+  // "--font-size-10",
+  "--font-size-20",
+  "--font-size-30",
+  "--font-size-40",
+  "--font-size-50",
+  "--font-size-60",
+  "--font-size-h1",
+  "--font-size-h2",
+  "--font-size-h3",
+  "--font-size-h4",
+  "--font-size-h5",
+  "--font-size-general-label",
+  "--font-size-paragraph",
+  "--font-size-subtext",
+];
+
+const FONT_WEIGHTS = ["--font-weight-very-light", "--font-weight-light", "--font-weight-normal", "--font-weight-bold"];
+
+const FONT_LINE_HEIGHTS = [
+  "--font-line-height-10",
+  // since "--font-line-height-20", "--font-line-height-30" and "--font-line-height-40" have the same value, we need to pick only one of them as "valid value", to allow autofix
+  // "--font-line-height-20",
+  // "--font-line-height-30",
+  "--font-line-height-40",
+  "--font-line-height-50",
+  "--font-line-height-60",
+  "--font-line-height-h1",
+  "--font-line-height-h2",
+  "--font-line-height-h3",
+  "--font-line-height-h4",
+  "--font-line-height-h5",
+  "--font-line-height-general-label",
+  "--font-line-height-paragraph",
+  "--font-line-height-subtext",
+];
+
+const SPACING_PROPS = [
+  "padding",
+  "padding-top",
+  "padding-bottom",
+  "padding-left",
+  "padding-right",
+  "padding-inline",
+  "padding-inline-end",
+  "padding-inline-start",
+  "padding-block",
+  "padding-block-end",
+  "padding-block-start",
+
+  "margin",
+  "margin-top",
+  "margin-bottom",
+  "margin-left",
+  "margin-right",
+  "margin-inline",
+  "margin-inline-end",
+  "margin-inline-start",
+  "margin-block",
+  "margin-block-end",
+  "margin-block-start",
+
+  "top",
+  "left",
+  "bottom",
+  "right",
```

Comment:
top, left, bottom and right can be used in component context or page context,  if we are in component context it might be logical to assume we will use spacing conventions, but its not 100% , in other contexts i am not sure this will be the case.

Reply:
hmm ok, so are you suggesting not to check the CSS props for `top`, `left` etc? I'm not sure I understand

Reply:
@3dyonic you convinced me - I removed these from the list :)